registry/storage/driver/testsuites/testsuites.go

```diff
@@ -269,7 +269,7 @@ func (suite *DriverSuite) TestWriteReadStreamsNonUTF8(c *check.C) {
 	suite.writeReadCompareStreams(c, filename, contents)
 }
 
-// TestWriteReadLargeStreams tests that a 5GB file may be written to the storage
```

Comment:
Root cause... no not sure. Reducing the size may have just skirted it.  

I'm on go 1.8.1, ubuntu 16.04lts, in a 12gig virtual box VM. Can reproduce the timeout issue with the following:
```
env GODEBUG=gctrace=1 go test -timeout=60m github.com/docker/distribution/registry/storage/driver/inmemory
```